src/utils/bannerUtility.ts

```diff
@@ -2,22 +2,23 @@ import { BannerConsumerProps } from 'HOCs/BannerProvider';
 
 export const ACCOUNT_VERIFICATION = 'Click here to verify your email and phone number to start booking!';
 
-export const showAccountVerificationBanner = async (isUserVerified: boolean, bannerActions: BannerConsumerProps['bannerActions'], bannerState: BannerConsumerProps['bannerState']): Promise<void> => {
-  const userVerified = isUserVerified && bannerState.showBanner;
-  const userUnverified = !isUserVerified && !bannerState.showBanner;
+export const showAccountVerificationBanner = (
+  userVerified: boolean,
+  bannerDispatch: BannerConsumerProps['bannerDispatch'],
+  bannerState: BannerConsumerProps['bannerState']): void => {
   const options = {
     message: ACCOUNT_VERIFICATION,
     background: 'secondary',
     textColor: 'white',
     to: '/account/verification',
   }
 
-  if (userVerified) {
-    await bannerActions.closeBanner();
+  if (userVerified && bannerState.show) {
+    bannerDispatch({ type: 'close' });
   }
 
-  if (userUnverified) {
-    await bannerActions.setBannerOptions(options);
-    await bannerActions.openBanner();
+  if (!(userVerified || bannerState.show)) {
```

Comment:
Is this more understandable? `!userVerified && !bannerState.show`
show the banner "if the user is not verified and the banner is not showing"

Reply:
Definitely is, fixing in next commit 🙏